internal/monitoring/README.md

```diff
@@ -0,0 +1,28 @@
+### **This wiki contains links of all the resources related to alerts and dashboards of the CI CD clusters**
+
+**Links to all AKS clusters used in CI CD**
+
+Below are the links to all the AKS clusters used in CI CD.
+We have 2 dev clusters and 2 prod clusters.
+
+* ci-prod-aks-eus - [link](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-prod-aks-eus-rg/providers/Microsoft.ContainerService/managedClusters/ci-prod-aks-eus/overview)
+
+* ci-prod-aks-mac-weu - [link](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-prod-aks-mac-weu-rg/providers/Microsoft.ContainerService/managedClusters/ci-prod-aks-mac-weu/overview)
+
+* ci-dev-aks-mac-eus - [link](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-dev-aks-mac-eus-rg/providers/Microsoft.ContainerService/managedClusters/ci-dev-aks-mac-eus/overview)
```

Comment:
Currently our CI/CD is still also deploying the 3P private preview chart (the pods named prom-dev-mac-prometheus-collector) in addition to the backdoor ama-metrics pods:

https://github.com/Azure/prometheus-collector/blob/16f02b39a60879e8d92fe4f5beb18de26ab3eaf5/.pipelines/azure-pipeline-build.yml#L411-L424

We will probably need to add a way to also use the addon chart with the CI/CD pipeline for the MAC clusters. I can help with this

Reply:
Sure. I have updated this in the latest commit.